src/experimental-layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -348,17 +344,15 @@ export default class SolidPolygonLayer extends Layer {
           geometry: new Geometry({
             drawMode: GL.TRIANGLES,
             attributes: {
-              vertexPositions: {size: 2, instanced: 1, value: new Float32Array([0, 1])},
-              nextPositions: {size: 3, instanced: 1, value: new Float32Array(3)},
-              nextPositions64xyLow: {size: 2, instanced: 1, value: new Float32Array(2)}
+              vertexPositions: {size: 2, isGeneric: 1, value: new Float32Array([0, 1])},
```

Comment:
These should probably be `true` instead of `1`, see comment about `instanced` divisors above.